tests/cache/tests.py

```diff
@@ -1144,6 +1144,30 @@ def test_incr_decr_timeout(self):
         cache.decr(key)
         self.assertEqual(expire, cache._expire_info[_key])
 
+    @override_settings(CACHES=caches_setting_for_tests(
+        BACKEND='django.core.cache.backends.locmem.LocMemCache',
+        OPTIONS={'MAX_ENTRIES': 30},
```

Comment:
Is there a special significance to 30? Would a smaller value like 10 work just as well?

Reply:
30 is a fine choice. It strikes a balance between a number too high (originally 300) and a number too low (e.g. 3).